src/test/java/teammates/test/pageobjects/GoogleLoginPage.java

```diff
@@ -84,24 +89,21 @@ private void handleApprovalPageIfAny() {
         waitForRedirectIfAny();
         boolean isPageRequestingAccessApproval = getPageSource().contains(EXPECTED_SNIPPET_APPROVAL);
         if (isPageRequestingAccessApproval) {
-            click(By.id("persist_checkbox"));
+            markCheckBoxAsChecked(browser.driver.findElement(By.id("persist_checkbox")));
             click(By.id("approve_button"));
             waitForPageToLoad();
         }
     }
 
     private void waitForRedirectIfAny() {
-        By metaRefreshBy = By.cssSelector("meta[http-equiv='refresh']");
-        if (isElementPresent(metaRefreshBy)) {
-            waitForElementToDisappear(metaRefreshBy);
-            waitForPageToLoad();
-        }
-
-        By noScriptBy = By.tagName("noscript");
-        if (isElementPresent(noScriptBy)) {
-            waitForTextContainedInElementAbsence(noScriptBy, "&lt;meta http-equiv=\"refresh\"");
-            waitForPageToLoad();
-        }
+        String loginRedirectUrl = Config.APP_URL + "/_";
```

Comment:
Not sure if I get it right: can you possibly mean `TestProperties.TEAMMATES_URL`?

Reply:
Oh yes oops 😅